src/app/guards/after-login-guard/after-login.guard.ts

```diff
@@ -0,0 +1,24 @@
+import { Inject, Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, CanActivate, Router, RouterStateSnapshot, UrlTree } from '@angular/router';
+import { Observable } from 'rxjs';
+import { LocalStorageToken } from 'src/app/localstorage.token';
+import { InternalRoutes } from 'src/app/utils/internal-routes';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AfterLoginGuard implements CanActivate {
+  constructor(
+    @Inject(LocalStorageToken) private localstorage : Storage,
+    private router : Router,
+  ) { }
+  canActivate(
+    route: ActivatedRouteSnapshot,
+    state: RouterStateSnapshot): Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | boolean | UrlTree {
+      if(this.localstorage.getItem('token')) {
+        this.router.navigateByUrl(InternalRoutes.HOME);
```

Comment:
I think, here we need to add admin route(/home/admin) instead of /home. @BurleVineeth

Reply:
What am I thinking is, instead of navigating to another route, why don't we stay in the same route. For that, currently in app.component.ts I am tracking the current Route and it is commented. So, please don't raise a comment on that. If you have any ideas or suggestions regarding this, please let me know.

Reply:
your point is correct. we can use the same route but need to access that route in the guard itself instead of the app component. For now , I am merging this PR.